examples/evidence-based-typescript.yml

```
@@ -0,0 +1,156 @@
+    path: 'src/widgets/**'
+    dependsOn: ['shared', 'entities', 'features']
+
+  - name: 'pages'
+    path: 'src/pages/**'
+    dependsOn: ['shared', 'entities', 'features', 'widgets']
+
+  - name: 'app'
+    path: 'src/app/**'
+    dependsOn: ['*']  # App layer can import from all
+
+rules:
```

YAMLlint (1.37.1)

[error] duplication of key "rules" in mapping

(key-duplicates)